test/e2e/kubevirt.go

```diff
@@ -747,6 +897,10 @@ passwd:
 
 		Expect(err).ToNot(HaveOccurred())
 
+		conntrackDumpingDaemonSet()
+		ovsFlowsDumpingDaemonSet("breth0")
+		iptablesDumpingDaemonSet()
```

Comment:
I wasn't thinking for these to be running constantly., I was thinking for them to enabled manually when troubleshooting was required.